lib/config/foldertype.go

```diff
@@ -9,15 +9,15 @@ package config
 type FolderType int
 
 const (
-	FolderTypeReadWrite FolderType = iota // default is readwrite
-	FolderTypeReadOnly
+	FolderTypeSendReceive FolderType = iota // default is readwrite
+	FolderTypeSendOnly
 )
 
 func (t FolderType) String() string {
 	switch t {
-	case FolderTypeReadWrite:
+	case FolderTypeSendReceive:
 		return "readwrite"
-	case FolderTypeReadOnly:
+	case FolderTypeSendOnly:
```

Comment:
Yeap, very confusing... I'm trying to help you guys here with way more than I planned to. I really don't want to spend a lot of time and nothing comes out of it.

I thought the purpose of this PR is to do the actual folder type name changes, since they've been confusing a lot of people. I don't think there's a lot of benefit to do this only partially. 
So far I was able to do all changes so that the user doesn't even need to do anything (API still pending). Obviously there will be new names displayed in the GUI, so there is user impact.

Of course I'm not familiar with the guidelines you follow what can and can't go into a minor release.
When are you planning to release 0.15?